### ISSUE TITLE
Bountysource integration tool not working [$5]

### ISSUE THREAD

**Issue by MichaelAtOz** [openscad/openscad#4369]
Tool was not flagging new issues, showed an error in BS settings (tools under your profile picture menu).
Showed as `This plugin is locked.` with a checked check-box.
So first attempt, uncheck it. Save.
See if that did it. ?

**Comment by MichaelAtOz** [comment]
We're all very observant. Broken since early Aug 2021...

**Comment by MichaelAtOz** [comment]
[Reported](https://github.com/bountysource/core/issues) a number of times at BS. No response from them.
I wonder if they are waiting for 
> Coming soon: A brand new website interface for an even better experience! 

Couldn't be much worse...

**Comment by MichaelAtOz** [comment]
I've changed to testing in my repo until that works.

**Comment by MichaelAtOz** [comment]
Everything works on my repo.
I removed the tool, and reinstalled, ~hour ago, doesn't seem to have done anything, but no locked error.

I just added a US$25 bounty for  #4314, expires in 1 month, **easy beer money**.
BS has not updated the Git issue.
I'll see tomorrow, it seems to be a cludgy interface.

**Comment by MichaelAtOz** [comment]
> Hmm, that link is the website repo. Maybe it got confused?

@t-paul 

Hmmm, missed that. I just tried to install on openscad/openscad  got: 'Tracker has already been taken'
Yep confused, I'll email support.

Do you think we should add others? Maybe MCAD, documentation (or openscad-documentation)?

> openscad 28 projects
> 		openscad/openscad  
> Installed 		openscad/openscad.github.com
> 		openscad/MCAD
> 		openscad/OpenCSG
> 		openscad/homebrew-tap
> 		openscad/statistics-scripts
> 		openscad/scad-utils
> 		openscad/list-comprehension-demos
> 		openscad/libtess2
> 		openscad/mxe
> 		openscad/svg-tests
> 		openscad/docker-openscad
> 		openscad/openscad-snap
> 		openscad/lib3mf-debian
> 		openscad/image-generator
> 		openscad/documentation
> 		openscad/openscad-advent-calendar-2019
> 		openscad/dev-support
> 		openscad/openscad-documentation
> 		openscad/openscad-advent-calendar-2020
> 		openscad/gsod-workflow-openscad
> 		openscad/openscad-lists
> 		openscad/openscad-libraries
> 		openscad/openscad-advent-calendar-2021
> 		openscad/discussion-test-repo
> 		openscad/openscad-registry
> 		openscad/openscad-wasm
> 		openscad/openscad-playground

**Comment by MichaelAtOz** [comment]
I installed the Tool to openscad/MCAD, got an acknowledgement message.
Created a test issue, but no joy.

I noticed on BS hitting 'save' on the Tool panel, takes forever, literally forever. 
Something bigger is broken. But it works in my repo...
Just have to wait on support.

**Comment by MichaelAtOz** [comment]
BS support reset everything. Installed tool on openscad/openscad, got confirmation message.
I'll have to add another test issue.

**Comment by MichaelAtOz** [comment]
OK! This issue just got the BS treatment. It may have triggered when I replied, as I did a refresh, without getting the BS.
Now waiting on new issue 4372 (didn't use a hash as I didn't want to impact the issue record).
Now going to play with some old issues, and add a bounty.

**Comment by MichaelAtOz** [comment]
I suspect it is back to normal, the old issues at Aug 2021 are getting BS-ified.
It has now done all old issues.

But 
a. New issue 4372 has not, 
b. New bounty on 4105 not flagged yet (was meant to be $30, but BS did the 'click on submit twice' bug, so $60 of **easy beer money**)

I'll check later. Please don't post to those issues just yet.

**Comment by MichaelAtOz** [comment]
Bounty on 4105 got set as it should.
Issue 4372 never got BS'ed. Closed it.
New test issue 4373 created 12:49.

Now over an hour and no BS on 4373.
@t-paul could you raise a test issue, I'm wondering if it does something different as I am now the registered BS Tool user.

**Comment by MichaelAtOz** [comment]
@t-paul scratch that. #4373 got the BS treatment.
Seems to be fully operations, if slow.
